### ISSUE TITLE
paths-kebab-case rule erroneously flagging non kebab-case parameter keys after v0.3.7 update

### ISSUE THREAD

**Issue by martinsirbe** [daveshanley/vacuum#337]
We've encountered an issue upon upgrading from version `v0.3.6` to `v0.3.7`. Specifically, the `paths-kebab-case` rule now throws warnings for parameter keys enclosed in curly brackets `{...}` if not specified in kebab case, rather than just for URL segments.

Would it be more appropriate for this rule to exclude parameter keys from its checks?

Here's the relevant error log:
```
[2023-09-08T13:37:16.507Z] Line / Column | Severity | Message                                                                             | Rule             | Path

[2023-09-08T13:37:16.507Z] (150:5)       | warning    | Path segments `{user_id}` do not use kebab-case                                  | paths-kebab-case | $.paths./users/{user_id}
```

For further clarification, I've created test cases which are currently failing due to the inclusion of parameter keys in the `paths-kebab-case` rule checks. You can view them here: [GitHub Test Cases](https://github.com/martinsirbe/vacuum/commit/4beee2116fa3ff1f8d3dcc84fe98c308357ed499).

**Comment by daveshanley** [comment]
Hi, yes this seems to be a regression from the change made here: 

https://github.com/daveshanley/vacuum/commit/7b4a65de7b6a13cc55f4d2fc8c5df13ad9bfa59c

I will re-visit the logic and tune it up.

**Comment by daveshanley** [comment]
This should be fixed in `v0.3.9`, would you be able to verify?

**Comment by martinsirbe** [comment]
This is now working as expected, thank you 👍
